renderer/brave_render_thread_observer.h

```diff
@@ -39,12 +41,14 @@ class BraveRenderThreadObserver
       blink::AssociatedInterfaceRegistry* associated_interfaces) override;
 
   // brave::mojom::BraveRendererConfiguration:
+  void SetInitialConfiguration(bool is_tor_process) override;
   void SetConfiguration(brave::mojom::DynamicParamsPtr params) override;
 
   void OnRendererConfigurationAssociatedRequest(
       mojo::PendingAssociatedReceiver<brave::mojom::BraveRendererConfiguration>
           receiver);
 
+  bool is_tor_process_;
```

Comment:
pls don't leave such things uninitialized. `= false`

Reply:
fixed and squashed in dadf783dc3aceac81b3435a58da571c95e126f61